inception-recommendation-api/src/main/java/de/tudarmstadt/ukp/inception/recommendation/api/recommender/RecommendationEngine.java

```diff
@@ -31,6 +31,12 @@
 import de.tudarmstadt.ukp.inception.recommendation.api.model.Recommender;
 
 public abstract class RecommendationEngine {
+	
+	public enum RecommendationEngineCapability {
```

Comment:
I would move this to a seperate class, as we use this field outside of `RecommendationEngine`.